Playground/RxSwiftExtPlayground.playground/Pages/Index.xcplaygroundpage/Contents.swift

```diff
@@ -49,6 +49,7 @@
  - **UIViewPropertyAnimator** [animate()](UIViewPropertyAnimator.animate) operator, returns a Completable that completes as soon as the animation ends.
  - **UIViewPropertyAnimator** [fractionComplete](UIViewPropertyAnimator.fractionComplete) binder, provides a reactive way to bind to `UIViewPropertyAnimator.fractionComplete`.
  - **UIScrollView** [reachedBottom](UIScrollView.reachedBottom) emits events when UIScrollView is scrolled to the bottom.
+ - [Observable.merge(with:)](mergeWith) merges elements from observable sequences into a single observable sequence.
```

Comment:
One minor thing - sorry :) 
Can you move it upwards, next to `zip(with):` ? We've been trying to keep RxCocoa operators at the bottom, I should document this somewhere... 

Same for README.md.

Reply:
Sure, done :)
Please don't forget to add that to CONTRIBUTING.md